src/main.rs

```diff
@@ -235,7 +238,21 @@ async fn run_server(addr: SocketAddr) -> anyhow::Result<()> {
     let pool = db::ClientPool::new();
     db::run_migrations(&*pool.get().await)
         .await
-        .context("database migrations")?;
+        .context("database migrations")?;  
+
+    // spawning a background task that will run the scheduled jobs
+    // every JOB_PROCESSING_CADENCE_IN_SECS
+    task::spawn(async move {
```

Comment:
Concerned about what would happen if the task dies. We should implement a logic here to recover from that? If so, do you have any suggestions? 

(we think a posible solution is match against what [JoinHandle](https://doc.rust-lang.org/nightly/std/thread/struct.JoinHandle.html) would have)

Reply:
I think you can do that with a tokio task just as well? Just await it's completion and re-launch it if it fails?

Reply:
Implemented a possible solution in https://github.com/rust-lang/triagebot/pull/1658/commits/2452683518a71627e685931a7de7953b5505601d.

We have the doubt if that's the best approach since we want to always re spawn the task in case of failure. Open to any other suggestion